### DISCUSSION TITLE
How many times has Tiger Woods won The Masters?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#177] [Galaxy Trivia] [answered]
**Hint:** 🕔

**Comment by thenithinbalaji** [comment]
5 times

**Comment by thuongtruong1009** [comment, marked as answer]
Tiger Woods has won The Masters golf tournament five times. He won the tournament in 1997, 2001, 2002, 2005, and 2019. His first victory at The Masters in 1997 was particularly notable, as he became the youngest player ever to win the tournament at just 21 years old, and he also set several records with his performance. Overall, Tiger Woods is widely considered one of the greatest golfers of all time, with numerous major tournament victories and other impressive accomplishments throughout his career.
